.env.test

```diff
@@ -8,3 +8,4 @@ POSTGRES_USER=root
 SLACK_BOT_USER_O_AUTH_ACCESS_TOKEN=SLACK_BOT_USER_O_AUTH_ACCESS_TOKEN
 SLACK_SIGNING_SECRET=SLACK_SIGNING_SECRET
 EMAIL_DOMAINS=my-domain.com,ext.my-domain.com
+GITLAB_URL=https://my.domain.com
```

Comment:
Suggestion: maybe an example like `https://my-git.domain.com` to be more clear on what is expected.